torch/_inductor/codegen/cpp.py

```diff
@@ -399,6 +399,19 @@ def _print_Abs(self, expr):
         assert len(expr.args) == 1
         return f"std::abs({self._print(expr.args[0])})"
 
+    def _print_Round(self, expr):
+        assert len(expr.args) == 1
+        return f"std::lrint({self._print(expr.args[0])})"
+
+    def _print_RoundDecimal(self, expr):
+        assert len(expr.args) == 2
+        number, ndigits = expr.args
+        assert isinstance(ndigits, sympy.Integer)
+        pre, post = f"1e{abs(ndigits)}", f"1e-{abs(ndigits)}"
+        if ndigits < 0:
+            pre, post = post, pre
+        return f"static_cast<double>(std::nearbyint({pre} * {self.paren(self._print(number))}) * {post})"
```

Comment:
This is a bit of an edge case, but I checked and you can have `RoundDecimal` on integers. For example, `round(123, -1) -> 120`.  Whereas this would promote it to float.